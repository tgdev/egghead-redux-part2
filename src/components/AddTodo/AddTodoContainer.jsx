import { connect } from 'react-redux';
import AddTodo from './AddTodo';

const AddTodoContainer = connect()(AddTodo);

export default AddTodoContainer;
