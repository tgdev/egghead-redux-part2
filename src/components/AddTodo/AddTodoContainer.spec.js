import React from 'react';
import { shallow } from 'enzyme';

import appStore from '../../store';

import AddTodoContainer from './AddTodoContainer';

describe('AddTodoContainer component', () => {

  const storeContext =  { context: { store: appStore } };

  it('renders AddTodoContainer correctly', () => {
    const wrapper = shallow(<AddTodoContainer />, storeContext);
    expect(wrapper).toMatchSnapshot();
  });

});
