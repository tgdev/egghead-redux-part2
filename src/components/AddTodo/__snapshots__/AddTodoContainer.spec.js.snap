// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTodoContainer component renders AddTodoContainer correctly 1`] = `
<AddTodo
  dispatch={[Function]}
/>
`;
